operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -1752,3 +1559,80 @@ func (rc *ReconciliationContext) CheckRollingRestart() result.ReconcileResult {
 
 	return result.Continue()
 }
+
+// ReconcileAllRacks determines if a rack needs to be reconciled.
+func (rc *ReconciliationContext) ReconcileAllRacks() (reconcile.Result, error) {
+	logger := rc.ReqLogger
+	logger.Info("reconcile_racks::Apply")
+
+	podList, err := rc.listPods(nil)
```

Comment:
I'm not sure about this - maybe we should filter by pods the operator manages? I know you filter down right under this, but there's some potential this will bring back dozens/hundreds of other pods we aren't responsible for

Reply:
actually what if we started off by passing in the cluster label filter here, so we initially get our clusterpods and then just grab the dc pods from there for the stuff that needs it. The dc filter already includes the cluster label, so it would cut down an extra filter step + avoid us grabbing anything that we didn't slap our cluster label on